source/Octopus.Tentacle.Contracts/ScriptServiceV3Alpha/ScriptCommandV3Alpha.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace Octopus.Tentacle.Contracts.ScriptServiceV3Alpha
+{
+    public abstract class ScriptCommandV3Alpha
```

Comment:
I create a base command to enforce that all commands must have the ScriptTicket property

Reply:
Yeah, that's fine. I don't feel strongly about it, so happy to defer.

Reply:
Addressed in c679e679